.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npm test
```

Comment:
Looks like only `npm test` pre-commit hook has been added. I think we want to `lint` and have a `commit` message check.